packages/react/src/hooks/useAuth.ts

```diff
@@ -15,28 +15,6 @@ export interface UseAuthResult {
   fetch?: () => void;
 }
 
-/**
- * React hook for Amplify Auth.
```

Comment:
Any reason for removing the useage comment? Seems useful.

Reply:
API is not public